dwave/system/samplers/clique.py

```diff
@@ -30,14 +30,15 @@
     from dimod import ScaleComposite
 
 from dwave.system.samplers.dwave_sampler import DWaveSampler, _failover
+from dwave.system.coupling_groups import coupling_groups
 
 __all__ = ['DWaveCliqueSampler']
 
 class _QubitCouplingComposite(dimod.ComposedSampler):
     """Composite that scales variables of a problem.
 
-    Checks whether the per qubit coupling range is violated for the qpu 
-    and rescale accordingly. Scales the variables of a binary quadratic 
+    Checks whether the per qubit (or per group) coupling range is violated for
+    the qpu and rescale accordingly. Scales the variables of a binary quadratic
```

Comment:
"the qpu and" --> "the QPU and"
"rescale" --> "rescales"

Reply:
✔️